deps/terminfo.version

```diff
@@ -0,0 +1,2 @@
+TERMINFO_VER := 2023.12.9
```

Comment:
To get right syntax highlighting:
```suggestion
# -*- makefile -*-
TERMINFO_VER := 2023.12.9
```

Reply:
The right syntax highlighting for who, Emacs users? Who even uses that? :trollface:

Reply:
For GitHub too, see for example https://github.com/JuliaLang/julia/blob/f0a2a7a0a9438433eb1ac824bd09627ab8fd0586/deps/curl.version#L1-L6

Reply:
GitHub? Who even uses that?